Sources/RMCLogger.cpp

```diff
@@ -12,6 +12,8 @@ namespace CTRPluginFramework {
         std::string procName;
         Process::GetName(procName);
 
+		titleID = Process::GetTitleID();
```

Comment:
tabs vs spaces

Reply:
GOD DAMMIT, I EVEN SET VSC TO USE SPACES FOR THIS PROJECT